charts/fpl-case-service/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: fpl-case-service
 apiVersion: v1
 home: https://github.com/hmcts/fpl-ccd-configuration
-version: 1.10.11
+version: 1.10.12
```

Comment:
i would update minor version instead of patch

Reply:
Yup, I will change it.